public/data/jadebanares.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Jade Bañares",
+  "type": "personal",
+  "bio": "Front-end Web Developer | Traditional Artist",
+  "avatar": "http://github.com/jade-arinal-banares.png",
+  "links": [
+    {
+      "name": "Check out my Portfolio",
+      "url": "https://jade-arinal-banares.vercel.app/",
+      "icon": "globe"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/notifications/",
```

Comment:
This link goes to the default LinkedIn feed. Perhaps you can add your profile link here?

Reply:
Oh I'm sorry, I forgot to change that. Will do :))